keyboards/vortex/m0110a/info.json

```
@@ -94,10 +94,11 @@
                     { "label": "L", "matrix": [7, 2], "x": 9.75, "y": 2 },
                     { "label": ";", "matrix": [8, 2], "x": 10.75, "y": 2 },
                     { "label": "'", "matrix": [8, 3], "x": 11.75, "y": 2 },
-                    { "label": "Enter", "matrix": [10, 4], "w": 2.25, "x": 12.75, "y": 2 },
-                    { "label": "Num4", "matrix": [11, 1], "x": 15.5, "y": 2 },
-                    { "label": "Num5", "matrix": [12, 1], "x": 16.5, "y": 2 },
-                    { "label": "Num6", "matrix": [13, 1], "x": 17.5, "y": 2 },
+                    { "label": "#", "matrix": [0, 2], "x": 12.75, "y": 2 },
+                    { "label": "Enter", "matrix": [10, 4], "w": 1.25, "x": 13.75, "y": 2 },
+                    { "label": "Num4", "matrix": [11, 1], "x": 15, "y": 2 },
+                    { "label": "Num5", "matrix": [12, 1], "x": 16, "y": 2 },
+                    { "label": "Num6", "matrix": [13, 1], "x": 17, "y": 2 },
```

I messed up my previous suggestion here.

```suggestion
                    { "label": "Num4", "matrix": [11, 1], "x": 15.5, "y": 2 },
                    { "label": "Num5", "matrix": [12, 1], "x": 16.5, "y": 2 },
                    { "label": "Num6", "matrix": [13, 1], "x": 17.5, "y": 2 },
```